### PR TITLE
build: put regular file at bazel-out by default

### PR DESCRIPTION
**Pull request by dt**
Release note: none.

### CODE REVIEW

**Review by ajwerner — APPROVED** [review]
:lgtm:

> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/74771#-:-MtFJB-72znfBFrE2eyr:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @irfansharif and @rickystewart)

File: bazel-out [hunk 1]
```diff
@@ -0,0 +1,3 @@
+this placeholder prevents bazel from writing a symlink here which points to its output dir.
+That symlink often causes tools that traverse the repo to get confused when they find lots of extra go files.
+The _bazel symlink points to the same thing, but go doesn't traverse into paths that start with _.
```
Comment (rickystewart) [inline bazel-out:3]: Nit: `_bazel` is not a symlink, but `_bazel/out` is.

Comment (dt) [reply bazel-out]: Fixed.

**Review by rickystewart — APPROVED** [review]

**Comment by dt** [comment]
TFTRs!

bors r+

**Pull request merged**